### PR TITLE
Extend logs, resolve empty category, add group fields

### PR DESCRIPTION
**Pull request by marcinsz84**
Extend logs,
resolve empty category array, 
add group leader fields to products with variables

### CODE REVIEW

File: doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php [hunk 1]
```diff
@@ -346,6 +349,31 @@ private function add_prices() {
...
+		if ( 'no' === $this->settings['split_variable'] ) {
+			return;
+		}
+
+		$id = $this->post->ID;
+		$product = wc_get_product( $id );
+		$parent = wc_get_product( $product->get_parent_id() ) ?: false;
+
+		if ( $product->is_type( 'variable' ) ) {
+			$this->fields[ 'df_group_leader' ] = 'true';
```
Comment (carlosescri) [inline doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php:367]: @marcinsz84 I think this field should be a boolean, not a string…

File: doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php [hunk 2]
```diff
@@ -346,6 +349,31 @@ private function add_prices() {
...
+			$this->fields[ 'df_grouping_id' ] = $id;
+		} else {
+			if ( $parent ) {
+				$this->fields[ 'df_group_leader' ] = 'false';
```
Comment (carlosescri) [inline doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php:371]: Same here…

**Review by carlosescri — COMMENTED** [review]
Just a couple of details, could you check?

File: doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php (hunk 1 above)
Comment (marcinsz84) [reply doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php]: Fixed

File: doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php (hunk 2 above)
Comment (marcinsz84) [reply doofinder-for-woocommerce/includes/data-feed/class-data-feed-item.php]: Fixed

**Pull request merged**